### PR TITLE
Feat/pm 752 career tab

### PR DESCRIPTION
**Pull request by dejwi**
[PM-752](https://playmakerpro.atlassian.net/browse/PM-752)

### Task Description

- added career tab

### Additional Notes (optional)

I'll add loader after creating resuable spinner component that they've added

### CODE REVIEW

File: components/profile/career/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useQuery } from '@tanstack/react-query'
+import { returnAfterTime } from 'dummy-api/helpers'
+import { profileCareerData } from 'dummy-api/profile/career'
+
+export const useProfileCareer = (id: string) =>
+  useQuery({
+    queryKey: ['profile-career', id],
+    queryFn: returnAfterTime(profileCareerData, 200),
```
Comment (mati-1) [inline components/profile/career/hooks.ts:8]: check query hook at my branch. We need more states, like loading, error itd, you need this to testing loader itd.

File: app/players/page.tsx [hunk 2]
```diff
@@ -12,9 +13,10 @@ const PlayerTestPage = () => (
     <ProfileHeader />
     <GradientedBg />
     <Tabs tabs={tabs} size="medium">
-      {tabs.map(tab => (
-        <p key={tab}>{tab}</p>
-      ))}
+      <span>Przegląd</span>
+      <ProfileCareer />
```
Comment (mati-1) [inline app/players/page.tsx:17]: put a carrer tab in these tabs and show me how to do it right. I will also put an overview tab after merging

**Review by mati-1 — COMMENTED** [review]
add these few things I need to see what your idea of ​​using the tabs

File: components/profile/career/hooks.ts (hunk 1 above)
Comment (dejwi) [reply components/profile/career/hooks.ts:8]: but you've already access to everything, it return whole useQuery object 😅
in useProfileHeader you just destructured the object and returned it structured again
and retry by default is 3

**Review by mati-1 — APPROVED** [review]

**Pull request merged**